src/emulator/types.ts

```diff
@@ -24,8 +34,10 @@ export const ALL_SERVICE_EMULATORS = [
   Emulators.PUBSUB,
 ];
 
+export const EMULATORS_SUPPORTED_BY_GUI = new Set([Emulators.DATABASE]);
```

Comment:
Should we change all these consts to `Set`s?

Reply:
I've looked into `ALL_SERVICE_EMULATORS` and found that we do use `.map` on it and put this into choices interactively, so I guess we still want deterministic ordering provided by the Array. So I actually changed the new const to `Array` instead to keep it consistent. (There's no performance issue on this one -- not on the critical path and it's a small constant anyway.)